packages/itwinui-css/backstop/tests/tag.html

```diff
@@ -632,18 +731,21 @@ <h2>Default tag containers</h2>
           </button>
         </span>
 
-        <span class="iui-tag">
+        <span
+          class="iui-tag"
+          role="option"
+        >
           <span
             class="iui-tag-label"
             title="Long tag"
           >
-            Long tag
+            Very long tag
```

Comment:
I think either this should be "Long tag" or the `title` and `aria-label` should be "Very long tag"

Reply:
That would explain [the unexpected test image change](https://github.com/iTwin/iTwinUI/pull/835#issuecomment-1343052369)!  Fixed.